articles/libraries/lock/v10/index.md

```diff
@@ -58,7 +57,7 @@ If you are using browserify or webpack to build your project and bundle its depe
 
 ## Usage
 
-### Implementing Lock
+### 1) Implementing Lock
```

Comment:
It's better if you use fullstops (`1.`) since this is the format we use at other docs as well.